tests/test_solr_prep.py

```diff
@@ -29,3 +30,63 @@ def test_legislative_session(bill,
     indexed_data = index.prepare(bill)
 
     assert indexed_data['legislative_session'] == prepared_session
+
+@pytest.mark.parametrize('session_identifier,prepared_session', [
+    ('2014', '7/1/2014 to 6/30/2015'),
+    ('2015', '7/1/2015 to 6/30/2016'),
+    ('2016', '7/1/2016 to 6/30/2017'),
+    ('2017', '7/1/2017 to 6/30/2018'),
+    ('2018', '7/1/2018 to 6/30/2019'),
+])
```

Comment:
What role does this play in your test?

Reply:
After doing a bit of reading, my understanding is that this decorator takes the tuples and turns them into parameters that can be used in the test. This decorator would take the tuple on line 35 and make `'2014'` be available in the test as `session_identifier`, and `'7/1/2014 to 6/30/2015'` as `prepared_session`, for example.

Before reading this, I thought that this was creating the values needed to create a the legislative session for a bill, but even then I wouldn't have needed all of them. After checking for and finding a `legislative_session` fixture, I understand that I don't need this decorator.